src/main/java/com/example/demo/model/UnUsedStaffCode.java

```diff
@@ -0,0 +1,31 @@
+package com.example.demo.model;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Documented
+@Constraint(validatedBy = {UnUsedStaffCodeValidator.class})
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+/* カスタム＠UnUsedStaffCodeに関するメソッド */
+
+public @interface UnUsedStaffCode {
```

Comment:
```suggestion
/* カスタム＠UnUsedStaffCodeに関するメソッド */
public @interface UnUsedStaffCode {
```
メソッドに対する説明なのであればコメントとメソッドの間に改行はないほうが何を指しているか明確だと思います。

Reply:
コメントを残す際にはわかりやすく伝わるように、気を付けていきます。
このファイルはカスタムバリデーションに関するファイルであるため、
不要になったため、削除いたしました。